packages/components/src/range-control/index.js

```diff
@@ -64,9 +60,12 @@ function RangeControl( {
 		// The input is valid, reset the local state property used to temporaly save the value,
 		// and call onChange with the new value as a number.
 		resetCurrentInput();
-		onChange( newNumericValue );
+		onChange( ( newValue === undefined || newValue === '' ) ?
```

Comment:
Is it even possible that `newValue` have a value of `undefined` here? I would think this could never possibly be the result of `event.target.value`.

Reply:
I checked this documentation page https://developer.mozilla.org/en-US/docs/Web/API/HTMLInputElement, value has type string and the docs have a not "Note: If the user enters a value different from the value expected, this may return an empty string.". I guess it is safe to conclude that undefined will never be the result of event.target.value. The code was updated.